pwiz_tools/Skyline/TestFunctional/DdaSearchTest.cs

```diff
@@ -182,9 +286,42 @@ private void TestAmandaSearch()
             RunUI(() =>
             {
                 Assert.IsTrue(importPeptideSearchDlg.CurrentPage == ImportPeptideSearchDlg.Pages.dda_search_settings_page);
-                importPeptideSearchDlg.SearchSettingsControl.PrecursorTolerance = new MzTolerance(15, MzTolerance.Units.ppm);
-                importPeptideSearchDlg.SearchSettingsControl.FragmentTolerance = new MzTolerance(25, MzTolerance.Units.ppm);
-                importPeptideSearchDlg.SearchSettingsControl.FragmentIons = "b, y";
+            });
+
+            SkylineWindow.BeginInvoke(new Action(() => importPeptideSearchDlg.SearchSettingsControl.SelectedSearchEngine = TestSettings.SearchEngine));
+
+            if (TestSettings.SearchEngine == SearchSettingsControl.SearchEngine.MSFragger)
+            {
+                var msfraggerDownloaderDlg = TryWaitForOpenForm<MsFraggerDownloadDlg>(2000);
+                if (msfraggerDownloaderDlg != null)
+                {
+                    RunUI(() =>
+                    {
+                        msfraggerDownloaderDlg.SetValues("Matt Chambers (testing download from Skyline)", "matt.chambers42@gmail.com", "UW");
+                        msfraggerDownloaderDlg.ClickAccept();

```

Comment:
The Accept button starts the download and closes the modal dialog after the download finishes. So yeah, I probably could switch back to using OkDialog. I had switched away from it because I was trying to catch the LongWaitDlg but that turned out to be impossible (and unnecessary) because it's finished when the modal dialog returns.